.changeset/healthy-eggs-nail.md

```diff
@@ -0,0 +1,5 @@
+---
+"@utrecht/select-css": minor
```

Comment:
Deze PR was bedoeld als bugfix van de React component, het lijkt me belangrijk daar ook een changelog entry voor te maken dan. Het is een bugfix, niet een feature, dus `patch` in plaats van `minor`.

Ik heb de andere Storybooks gecontroleerd, en dit was ook kapot [in de Vue.js Select component](https://nl-design-system.github.io/utrecht/storybook-vue/index.html?path=/docs/vue-select--docs), dus ook die moet genoemd worden. De Select component staat niet in de Storybook van Angular en Web Components, dus die heb ik niet genoemd.

Zoals eerder beschreven in de documentatie over CSS fixes, moet je ook `component-library-css` noemen voor dezelfde fix.

```suggestion
"@utrecht/component-library-css": patch
"@utrecht/component-library-react": patch
"@utrecht/component-library-vue": patch
"@utrecht/select-css": patch
```